clients/src/test/java/org/apache/kafka/common/metrics/SanitizerTest.java

```diff
@@ -21,6 +21,7 @@
 
 import java.io.UnsupportedEncodingException;
 
+import org.apache.kafka.common.utils.Sanitizer;
```

Comment:
Perhaps move this unit test to `utils` where `Sanitizer` was moved to?

Reply:
Can you remove this import since it is no longer required since `SanitizerTest` is in the same package - it is causing checkstyle failure.